py/server/deephaven/table.py

```diff
@@ -557,7 +557,7 @@ def columns(self) -> List[Column]:
     @property
     def meta_table(self) -> Table:
```

Comment:
@jmao-denver any reason to consider renaming the python method to `meta` or not?